actions/docs-verifier/src/BuildVerifier.IO.Abstractions/BaseConfigurationReader.cs

```diff
@@ -31,6 +31,16 @@ public abstract class BaseConfigurationReader<TConfigurationFile>
         if (_fileExists is null)
         {
             _fileExists = File.Exists(ConfigurationFileName);
+
+            // Try one level deeper.
```

Comment:
Instead of just checking the `docs` sub-folder, would it be better (although slower) to use `Directory.GetFiles` and set the `EnumerationOptions` to check sub-foders?